docs/concepts/work-pools.md

```diff
@@ -0,0 +1,273 @@
+---
+description: Prefect work pools route deployment flow runs to agents. Prefect agents poll work pools for new runs to execute.
+tags:
+    - work pools
+    - agents
+    - orchestration
+    - flow runs
+    - deployments
+    - schedules
+    - concurrency limits
+---
+
+# Agents and Work Pools
+
+Agents and work pools bridge the Prefect _orchestration environment_ with a user’s _execution environment_. When a [deployment](/concepts/deployments/) creates a flow run, it is submitted to a specific work pool for scheduling. Agents running in the execution environment poll their respective work pools for new runs to execute.
+
+Work pools are automatically created whenever they are referenced by either a deployment or an agent. For most applications, this automatic behavior will be sufficient to run flows as expected. However, advanced users will find that work pools can be configured to enable a greater degree of control over work delivery.
+
+To run deployments, you must configure at least one agent (and its associated work pool):
+
+1. [Start an agent](#starting-an-agent)
+2. [Configure a work pool](#work-pool-configuration) (optional)
+
+!!! tip "Agent role has changed from Prefect 1"
+    Work pools are a new concept. The role of agents has changed from their implementation in Prefect 1. If you're familiar with that model, please take some time to understand the new work pool/agent model. It requires a little more setup, but offers much greater control and flexibility with how deployments are executed.
+
+    Key changes: 
+    
+    - Work pools contain all the logic about what flows run and how. Agents just pick up work from pools and execute the flows.
+    - There is no global agent that picks up deployed work by default. You must configure an agent with a specific work pool.
+
+## Agent overview
+
+Agent processes are lightweight polling services that get scheduled work from a [work pool](#work-pool-overview) and deploy the corresponding flow runs. 
+
+It is possible for multiple agent processes to be started for a single work pool, or for one agent to pull work from multiple work pools. Each agent process sends a unique ID to the server to help disambiguate themselves and let users know how many agents are active.
+
+### Agent options
+
+Agents are configured to pull work from one or more work pools. If the agent references a work pool that doesn't exist, it will be created automatically.
```

Comment:
```suggestion
Agents are configured to pull work from one or more work pool queues. If the agent references a work queue that doesn't exist, it will be created automatically.
```